app/components/search/body.tsx

```diff
@@ -0,0 +1,14 @@
+import { useSearchParams } from "next/navigation";
+
+const SearchBody = () => {
```

Comment:
Nitpick: What do you think about "SearchContent" rather than "SearchBody"?

Reply:
fine with me